VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/quick-links/quick-links.component.html

```diff
@@ -9,7 +9,7 @@ <h1 class="govuk-heading-xl govuk-!-margin-top-8">{{ 'quick-participant-landing-
       <label class="govuk-label" for="full-name">
         {{ 'quick-participant-landing-page.name-label' | translate }}
       </label>
-      <input formControlName="name" class="govuk-input" id="full-name" name="full-name" type="text" spellcheck="false" autocomplete="name">
+      <input formControlName="name" class="govuk-input full-name-text-input" id="full-name" name="full-name" type="text" spellcheck="false" autocomplete="name">
```

Comment:
Is there not an existing class in gov uk that we can use for this?

Reply:
thats what I was discussing with Neil, it doesnt appear to be one, the one provided in the ticket wouldnt look good in ipad

Reply:
.govuk-grid-column-one-half

Reply:
The class I was attempting to use was this one: govuk-input govuk-!-width-one-half, and it doesnt look good in ipad, the other class Neil wanted me to try out with didnt have media query so was using a fixed width for all media devices

Reply:
I didn't see your comment regarding why. This class should already have responsiveness for smaller devices.